how-to/how-to-use-rootless-kata-containers-with-podman.md

```diff
@@ -0,0 +1,275 @@
+> Rootless support for Kata should be available and functional in upcoming 1.9.2 and 1.10.0-alpha1 releases.
+
+## Configuration
+
+Now that you have installed Kata Containers and Podman, you need to configure
+them for rootless execution.
+
+### Disable SELinux
+
+> **Warning:**
+>
+> You should understand the security implications before disabling SELinux.
+
+If SELinux is installed and enabled, it needs to be disabled with the
+following command (Kata Containers
+[does not support SELinux](https://github.com/kata-containers/documentation/blob/master/Limitations.md#selinux-support)).
+
+> **Warning:**
+> The following command might differ depending on the distro you use:
+
+```bash
+$ [ -f /etc/selinux/config ] && sudo sed -i 's/^SELINUX=.*/SELINUX=disabled/g' /etc/selinux/config
+```
+
+### Add user to KVM group
+
+If running a KVM based hypervisor, add the user running the workload to the KVM group:
+
+```bash
+$ sudo usermod -a -G kvm $USER
+```
+> **NOTE:**
+>
+> `kvm` should be the group owning the device node `/dev/kvm` on most distros.
+> Make sure permissions on `/dev/kvm` are as shown below:
+> ```
+> $ ls -la /dev/kvm
+> crw-rw---- 1 root kvm 10, 232 Nov 11 20:28 /dev/kvm
+> ```
+> If the group owner happens to be `root`, you may need to create a system group `kvm` and
+> change permissions for `/dev/kvm` as shown above.
+> ```
+> $ sudo addgroup --system kvm
+> $ sudo chown root:kvm /dev/kvm
+> $ sudo chmod g+rw /dev/kvm
+> ```
+ 
+### Reboot
+
+Reboot the system for the changes to take effect (when you disable SELinux you
+must reboot, while logging out and back in is enough to have that user joining
+the `kvm` group).
+
+Verify the configuration is correct:
+
+- If installed, disable SELinux:
+  ```bash
+  $ getenforce
+  Disabled
+  ```
+
+- The user should be in the `kvm` group:
+
+  ```
+  $ groups | grep -ow kvm
+  kvm
+  ```
+
+### Setup Kata configuration files
+
+If the Kata `configuration.toml` file does not exist in `/etc`, create it:
+
+```bash
+$ sudo mkdir -p /etc/kata-containers/
+$ sudo install -D -o ${USER} -g root -m 0640 /usr/share/defaults/kata-containers/configuration.toml /etc/kata-containers/
+```
+
+Or, if the file exists, but is not readable by the user:
+
+```bash
+$ sudo chown -R a+r /etc/kata-containers/
+```
+
+### Disable `vhost-net`
+
+Disable `vhost-net` in the Kata configuration file, by commenting out the
+`disable_vhost_net` line:
+
+```bash
+$ sudo sed -i -e 's/^#disable_vhost_net = true/disable_vhost_net = true/' /etc/kata-containers/configuration.toml
+```
+
+The above step is required in Kata version 1.10.0-alpha1. In future releases, this should
+no longer be required as Kata runtime should handle this automatically for rootless case.
+This does mean you will get slightly degraded network performance.
+
+### Modify the Kata images permissions
+
+The current user needs to have read and write permissions for the kata image, write permissions are required
```

Comment:
The comma after image should be a period, and write should start a new sentence.